openmdao/core/notebook_utils.py

```diff
@@ -40,18 +40,18 @@ def _get_object_from_reference(reference):
             right.append(split.pop())
     if module:
         for entry in reversed(right):
-            module = getattr(module, entry)
-    return module
+            obj = getattr(module, entry)
```

Comment:
This won't work because it's doing a getattr repeatedly using 'module' which now never updates.

Reply:
Missed that commit. Changed